webapp/channels/src/components/widgets/tag/tag.tsx

```diff
@@ -9,7 +9,7 @@ import styled, {css} from 'styled-components';
 import glyphMap from '@mattermost/compass-icons/components';
 import type {IconGlyphTypes} from '@mattermost/compass-icons/IconGlyphs';
 
-export type TagVariant = 'info' | 'success' | 'warning' | 'danger';
+export type TagVariant = 'info' | 'success' | 'warning' | 'danger' | 'dangerDim';
```

Comment:
@abhijit-singh I don't see this variant in https://compass.mattermost.com/29be2c109/p/46ae87-tag. Should that be included there?

Reply:
Yes dim version of all/some needs to be updated

Reply:
Yep, I have it on my list to add that variation to the docs.